lib/aggregate/attribute/hash.rb

```diff
@@ -75,7 +75,7 @@ def should_store_hash_as_json
         if options.key?(:store_hash_as_json)
           options[:store_hash_as_json]
         else
-          options.dig(:aggregate_db_storage_type) != :elasticsearch
+          true
```

Comment:
to avoid an explicit `true` return, we can rework this method
to be
```rb
options.exclude?(:store_hash_as_json) || options[:store_hash_as_json]
```

Reply:
or even just `!!options[:store_hash_as_json]`

Reply:
`!!options[:store_hash_as_json]` would convert `nil` to `false`, so I'll defer to the original implementation. Unless you meant merging both comments...?

Reply:
ah right. So `!!options[:store_hash_as_json]` wouldn't work 🤦‍♂ 

but the other one liner would